manager/subscription/heartbeat_manager.py

```diff
@@ -58,14 +58,18 @@ async def query_commander(cls):
 
             This is what the `commander_heartbeat_task` does
             """
-            heartbeat_url = f"http://{os.environ.get('COMMANDER_HOSTNAME')}:{os.environ.get('COMMANDER_PORT')}/heartbeat"
+            heartbeat_url = "http://"
+            +os.environ.get("COMMANDER_HOSTNAME")
+            +":"
+            +os.environ.get("COMMANDER_PORT")
+            +"/heartbeat"
             while True:
                 try:
                     # query commander
                     resp = requests.get(heartbeat_url)
-                    timestamp = resp.json()['timestamp']
-                    #get timestamp
-                    cls.set_heartbeat_timestamp('Commander', timestamp)
+                    timestamp = resp.json()["timestamp"]
+                    # get timestamp
+                    cls.set_heartbeat_timestamp("Commander", timestamp)
```

Comment:
Apparently single quotes make a difference jejejejeje

Reply:
jeje it seems, this are automatic black formatting fixes